hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/clustering/run/strategy/SingleSparkJobExecutionStrategy.java

```diff
@@ -79,94 +54,25 @@ public SingleSparkJobExecutionStrategy(HoodieTable table, HoodieEngineContext en
 
   @Override
   public HoodieWriteMetadata<HoodieData<WriteStatus>> performClustering(final HoodieClusteringPlan clusteringPlan, final Schema schema, final String instantTime) {
-    JavaSparkContext engineContext = HoodieSparkEngineContext.getSparkContext(getEngineContext());
     final TaskContextSupplier taskContextSupplier = getEngineContext().getTaskContextSupplier();
     final SerializableSchema serializableSchema = new SerializableSchema(schema);
     final List<ClusteringGroupInfo> clusteringGroupInfos = clusteringPlan.getInputGroups().stream().map(ClusteringGroupInfo::create).collect(Collectors.toList());
 
-    String umask = engineContext.hadoopConfiguration().get("fs.permissions.umask-mode");
```

Comment:
None of this logic is valid, and the class hasn't been used anywhere before, so I refactor this part of the code